samples/README.md

```diff
@@ -2,7 +2,7 @@
 
 Samples are a collection of small and simple apps that demonstrate how to use the library.
 
-To run the samples, we recommend to use go 1.16 or later. You will also need docker when using Fabric.
+To run the samples, we recommend to use go 1.17 or later. You will also need docker when using Fabric.
```

Comment:
why 1.17 and not 1.18? If the docs says 1.17, be should better have that version tested via CI.

Reply:
addressed with a new PR that I just merged.